src/style.css

```diff
@@ -0,0 +1,89 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url("./fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 900;
+  src: url("./fonts/Avenir-Heavy.ttf");
+}
+
+.body {
+  font-family: Avenir, Arial, sans-serif;
+  font-size: 16px;
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+.form {
+  margin-top: 20px;
+  position: relative;
+}
+
+.img {
+  position: absolute;
+}
+
+.img-lg {
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  left: 26px;
+}
+
+.img-sm {
+  width: 11px;
+  height: 11px;
+  top: 15px;
+  left: 13px;
+}
+
+.search {
+  width: 100%;
+  color: #3d4e61;
+  font-family: inherit;
+  font-weight: 300;
```

Comment:
inherit

Reply:
Thank you for review! I guess work with pixels was little bit painful so I've made couple mistakes after that. All of remarks make sense.